minter-suite/minter-suite-supplemental.md

```diff
@@ -139,3 +139,49 @@ console.log(
   merkleProof
 );
 ```
+
+---
+
+## Pre-Assigning Token Hash Seeds
+
+The shared randomizer `SharedRandomizerV0` enables an artist to pre-assign token hash seeds for tokens on a project. This allows an artist to assign a specific hash seed to a token before it is minted, and then mint the token with that hash seed at a later time.
```

Comment:
would be nice in an info callout or some other highlight

Reply:
updated in [977abdc](https://github.com/ArtBlocks/artblocks-docs/pull/312/commits/977abdc26fbfcd51f847cc118c13ce7bac59b0fa)